l2-contracts/contracts/bridge/L2StandardDeployer.sol

```diff
@@ -30,35 +30,35 @@ contract L2StandardDeployer is IL2StandardDeployer, Initializable {
     /// @dev Bytecode hash of the proxy for tokens deployed by the bridge.
     bytes32 internal l2TokenProxyBytecodeHash;
 
-    mapping(bytes32 assetInfo => address tokenAddress) public override tokenAddress;
+    mapping(bytes32 assetInfo => address tokenAddress) public override tokenAddress; // ToDo: why here token address and in shared bridge asset address?
```

Comment:
In the SharedBridge we work for general assets, here we only work for tokens. It is the same thing however, I'm not sure which naming is cleaner.